packages/oui-tooltip/src/tooltip.html

```diff
@@ -0,0 +1,13 @@
+<div class="oui-tooltip oui-message" role="tooltip">
+  <span aria-hidden="true"
+    ng-class="{
```

Comment:
Is this part can be simplified in a method?
It will be simplified when if it remains only top|bottom|left|right , but still...

Another point: one-time binding maybe?